gui/modules/upgrade/__init__.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import re
+
+class UpdateUtils():
+
+    @staticmethod
+    def get_repo_version_list():
+        url = "https://api.github.com/repos/dOpensource/dsiprouter/releases"
+        payload = {}
+        headers = {
+            'Accept': 'application/vnd.github+json',
+            'Authorization': 'Bearer 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',
```

Comment:
should not use hard-coded credentials  
listing the versions should not require auth